test/util.py

```diff
@@ -147,6 +154,11 @@ def fuse_test_marker():
 
 # Try to use local fusermount3
 os.environ['PATH'] = '%s:%s' % (pjoin(basename, 'util'), os.environ['PATH'])
+# Put example binaries on PATH
+os.environ['PATH'] = '%s:%s' % (pjoin(basename, 'example'), os.environ['PATH'])
+
+# Enable shared libraries to be resolved from build directory
+os.environ['LD_LIBRARY_PATH'] = pjoin(basename, 'lib')
```

Comment:
This should not be necessary, I think. Why is it needed now? The examples are linked against libfuse3 and do not need an specific LIBRARY_PATH

Reply:
You are right - I must have mistakenly added this when trying to get mount.fuse3 to successfully execute the hello test binary. Removing.